README.md

```diff
@@ -80,34 +96,35 @@ mkdir -p /tmp/rstudio-server
 
 Then launch the server similar to the docker command but with additional configuration options required:
```

Comment:
I'd suggest you avoid potential confusion here and not mention docker here. Remove this line.

Reply:
ok DONE